src/main/cpp/hw/DragonSparkMax.cpp

```diff
@@ -191,12 +205,15 @@ bool DragonSparkMax::IsForwardLimitSwitchClosed()
 
 bool DragonSparkMax::IsReverseLimitSwitchClosed()
 {
+    if (m_spark->GetReverseLimitSwitch(m_reverseType).Get())
```

Comment:
If we have m_reverseLimitSwitch, why don't we use it?

Reply:
Agreed, changed.